### DISCUSSION TITLE
Is there a way to use "EventLists" on terminal with hyprctl ?

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#3344] [General]
I wanted to auto close skype upon start up, because this starts with a blank page.

exec-once = skypeforlinux && closewindow skypeforlinux 

I wanted something similar to that, i know you can bind closewindow, but can i do it on start up?

**Comment by MightyPlaza** [comment]
hyprctl dispatch?

**Reply by ghost** [reply]
It worked thank you
